include/aspect/plugins.h

```diff
@@ -181,7 +181,7 @@ namespace aspect
          * need not worry whether we try to add something to this list before
          * the lists's constructor has successfully run
          */
-        static std::list<PluginInfo> *plugins;
+        static std::unique_ptr<std::list<PluginInfo>> plugins;
```

Comment:
Does the comment above still apply regarding initialization order? unique_ptr does have a constructor.